src/EditorFeatures/Core/Implementation/InlineRename/CommandHandlers/RenameCommandHandler_LineStartEndHandler.cs

```diff
@@ -92,16 +92,22 @@ private bool HandleLineStartOrLineEndCommand(ITextBuffer subjectBuffer, ITextVie
                         return false;
                     }
 
+                    var newPointInView = view.BufferGraph.MapUpToBuffer(newPoint, PointTrackingMode.Negative, PositionAffinity.Successor, view.TextBuffer);
```

Comment:
Please add a comment explaining your choice.

I would imagine that point tracking mode doesn't matter if you're not tracking between different versions. You'll probably want to think carefully about PositionAffinityMode though.